From 7950a07afaa44eef17b1cab9c036953036878d73 Mon Sep 17 00:00:00 2001
From: Wu Chia Chuan <chia.chuan.wu@intel.com>
Date: Wed, 28 Oct 2015 16:43:47 +0800
Subject: [PATCH]  fix kernel patch

Signed-off-by: Wu Chia Chuan <chia.chuan.wu@intel.com>
---
 quickassist/qat/Makefile |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/quickassist/qat/Makefile b/quickassist/qat/Makefile
index 319665d..36db6e6 100755
--- a/quickassist/qat/Makefile
+++ b/quickassist/qat/Makefile
@@ -6,13 +6,13 @@ export CONFIG_CRYPTO_DEV_QAT_C62X=m
 export QAT_UIO?=y

 ifeq ($(KERNELRELEASE),)
-KDIR ?= $(INSTALL_MOD_PATH)/lib/modules/$(shell uname -r)/build
-ifneq ($(shell if [ -e $(KDIR)/include/config/auto.conf ]; then echo 1; fi),1)
+KDIR ?= $(KERNEL_SOURCE_ROOT)
+ifneq ($(shell if [ -e $(KERNEL_BUILDDIR)/include/config/auto.conf ]; then echo 1; fi),1)
   $(error ERROR: Kernel header files not found.  Install the appropriate \
     kernel development package necessary for building external kernel modules \
     or run 'make oldconfig && make modules_prepare' on kernel src to fix it)
 endif
-include $(KDIR)/include/config/auto.conf
+include $(KERNEL_BUILDDIR)/include/config/auto.conf
 INSTALL_FW_PATH ?= $(INSTALL_MOD_PATH)/lib/firmware

 default: modules
--
1.7.9.5
